terraform/aws/components/nhais/data_secrets.tf

```diff
@@ -34,6 +34,10 @@ data "aws_secretsmanager_secret" "nhais_mesh_endpoint_cert" {
   name = "nhais_${var.environment}_mesh_endpoint_cert"
 }
 
+data "aws_secretsmanager_secret" "nhais_mesh_sub_ca" {
+  name = "nhais_${var.environment}_mesh_sub_ca"
```

Comment:
does not exist in secrets manager

Reply:
ptl: we need to add the real CA cert for INT
build1: use the CA cert for OpenTest
vp: can be anything / blank, we'll use opentest and disable cert validation